.gitignore

```diff
@@ -44,6 +44,7 @@ incoming_chat.txt
 locobot/test/test_assets/perception_handlers/
 locobot/agent/dashboard_data.db*
 nsp_outputs.csv
+agents/locobot/interaction_loggings.json
```

Comment:
Are there more file outputs that you don't want to commit? This can match `*.json` for example

Reply:
I do not think so, since this is the only new file I introduced to the codebase